applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -59,6 +61,7 @@ def Execute(self):
 
```

Comment:
The comment says that this is done to find the minimal X. I feel that it is misleading...

Reply:
You are right, it is misleading. This process is cleaned up and better commented in https://github.com/KratosMultiphysics/Kratos/pull/4659. You can take a look in there and if there are still open questions I would gladly answer them. Also you can take a look at https://github.com/KratosMultiphysics/Kratos/pull/4661#issuecomment-483535939